blobfuse/src/BlobfuseGcCache.cpp

```diff
@@ -205,6 +205,10 @@ void gc_cache::run_gc_cache()
                             // Failed to acquire the lock for some other reason.  We close the open fd, and continue.
                             syslog(LOG_ERR, "Did not clean up file %s from file cache because we failed to acquire the flock for an unknown reason, errno = %d.\n", mntPath, errno);
                         }
+                        if (permissionsoverwritten == true)
+                        {
+                            fchmod(fd, originalpermissions);
```

Comment:
There is build failure due to this statement :

/home/vsts/work/1/s/blobfuse/src/BlobfuseGcCache.cpp:210:35: error: ‘originalpermissions’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
                             fchmod(fd, originalpermissions);

we need to initialize this variable in the begining.